install/roles/dovecot/files/access-check-whitelist.sh

```diff
@@ -53,7 +53,7 @@ fi
 # Check if the IP address is whitelisted
 whitelisted="0"
 if [ -r "$whitelistFile" ]; then
-    whitelisted=$(grepcidr -x -c "$IP" "$whitelistFile")
+    whitelisted=$(echo "$IP" | grepcidr -x -c -f "$whitelistFile")
```

Comment:
Maybe keep this change for another commit.

Reply:
idem

Reply:
Removed, it will be in another PR.